src/collections/_documentation/workflow/integrations/gitlab.md

```diff
@@ -56,4 +56,84 @@ Sentry owner or manager permissions are required to install this integration.
+    
+## Commit Tracking
+
+Commit tracking allows you to hone in on problematic commits. With commit tracking, you can better isolate what went out and what might be problematic, by leveraging information from releases like tags and metadata.
+
+{% capture __alert_content -%}
+For commit tracking to work, you must click **configure** in Configure GitLab step 8.
+{%- endcapture -%}
+{%- include components/alert.html
+  title="Note"
+  content=__alert_content
+  level="warning"
+%}
+
+The image below displays how a release can shed light on who exactly made commits, which projects were affected, and which deploys were impacted.
+
+[{% asset gitlab/last-commit-in-releases.png alt="Dashboard with last commit highlighted" %}]({% asset gitlab/last-commit-in-releases.png @path %})
+
+When you investigate deeper into that commit, you can leverage information from metadata like tags.
+
+[{% asset gitlab/highlighting-tags.png alt="Issue detail highlighting tags" %}]({% asset gitlab/highlighting-tags.png @path %})
+
+Broadly, this lets you isolate problems in order to quickly see what was 'committed' and what might be problematic.
+
+Learn more about how to configure [commit tracking]({%- link _documentation/workflow/releases.md -%}#link-repository).
+
+## Suspect Commit
+
+Once you are tracking the commits, the 'suspect commit' is the commit that likely introduced the error.
+
+One special benefit of using Sentry's Commit Tracking is the ability to know the suspect commit that likely caused the error, with a suggested plan of action for how to rectify the error. For example, after pinpointing the suspect commit, you can also identify the developer who made the commit and assign them the task of fixing the error. 
+
+[{% asset gitlab/highlighting-suspect-commits.png alt="Issue detail highlighting suspect commits" %}]({% asset gitlab/highlighting-suspect-commits.png @path %})
+
+Here is where you can find info for [suspect commit setup]({%- link _documentation/workflow/releases.md -%}#link-repository).
+
+## Resolve via Commit or PR
+
+Once you are sending commit data, you can start resolving issues by including `fixes <SHORT-ID>` in your commit messages. You might want to type something in the commit like: "this fixes Sentry-AB12" or "Fixes MyApp-317". The keyword to include is **fixes**. You can also resolve issues with pull requests by including `fixes <SHORT-ID>` in the title or description. This will automatically resolve the issue in the next release. 
+
+## Troubleshooting
+
+FAQ:
+- Do I need to whitelist the IP address?
+    - If you are using Sentry on-premise, then yes, you do.
+    - For more details, checkout our [IP Ranges docs]({%- link ip-ranges.md -%}).
+- Do you support subgroups?
+    - Organizations can sit within organizations, but we don't support subgroups for this unless you use GitLab version 11.6 or higher.
+- Why is my integration not working under groups?
```

Comment:
Can we change this to: 
- My repositories are hosted under my user account, not a group account. Can I still use this integration?
- Unfortunately not. The GitLab integration only works for repositories that are hosted under group accounts.